docs/en/observability/whats-new.asciidoc

```diff
@@ -11,4 +13,278 @@ https://www.elastic.co/blog/elastic-observability-7-6-0-released[7.6] |
 https://www.elastic.co/blog/elastic-observability-7-5-0-released[7.5] |
 https://www.elastic.co/blog/elastic-observability-update-7-4-0[7.4]
 
-coming[7.10.0]
+[discrete]
+== Give {minor-version} a try
+
+Try {minor-version} now by deploying {es} and {kib} on
+https://www.elastic.co/cloud/elasticsearch-service/signup[Elastic Cloud] or
+by https://www.elastic.co/start[downloading them].
+
```

Comment:
Can you also add a tagged region that starts here? We need to use this content to replace the APM section in the stack docs. See https://www.elastic.co/guide/en/elastic-stack/7.9/elastic-stack-highlights.html

Reply:
I added a `tag::710-new[]` region. Would you double check to ensure you want all of the features included? Thanks!

Reply:
Thank you! Looks good, but I apologize, I should have mentioned to name it generically

Reply:
No problem. Thanks!